### DISCUSSION TITLE
Access git logs in build process

### DISCUSSION THREAD

**Discussion by egoist** [vercel/vercel#4101] [General] [answered]
__Is it possible to get `git logs` during the build process on Now?__ That would be useful for retrieving the last modification time of repository files, currently you have to manually do it locally in a post commit hook like what this repo does: https://github.com/zeit/docs/blob/f45d4df77f443e14c80d91c94ed400b445c23db0/scripts/update-mdx-meta.js#L44

**Comment by styfle** [comment, marked as answer]
Currently this is not possible because the `.git` directory is not deployed. The only git metadata available today is provided as [system environment variables](https://zeit.co/docs/v2/build-step#system-environment-variables).

In the future, we are looking at providing the full `.git` directory during build time to solve problems like the one you mentioned.

Here's a list of what you can use today:

![env vars](https://user-images.githubusercontent.com/229881/79797844-fffcaf00-8325-11ea-938e-6bc00c4b2cb6.png)

**Comment by Benbinbin** [comment]
The `.git` log file accessibility makes sense [at some point](https://github.com/nuxt/content/pull/1839#issuecomment-1489624989)
